ph5/clients/ph5availability.py

```diff
@@ -540,15 +551,19 @@ def get_availability(self, station='*', location='*',
                         ph5_start_epoch = st['deploy_time/epoch_l']
                         ph5_stop_epoch = st['pickup_time/epoch_l']
                         ph5_sample_rate = st['sample_rate_i']
-                        ph5_multiplier = st['sample_rate_multiplier_i']
-                        Das_t = self.ph5.query_das_t(
-                            ph5_das,
-                            chan=channum,
-                            start_epoch=ph5_start_epoch,
-                            stop_epoch=ph5_stop_epoch,
-                            sample_rate=ph5_sample_rate,
-                            sample_rate_multiplier=ph5_multiplier,
-                            check_samplerate=False)
+                        ph5_multiplier = self.get_srm(st)
+                        try:
+                            Das_t = self.ph5.query_das_t(
+                                ph5_das,
+                                chan=channum,
+                                start_epoch=ph5_start_epoch,
+                                stop_epoch=ph5_stop_epoch,
+                                sample_rate=ph5_sample_rate,
+                                sample_rate_multiplier=ph5_multiplier,
+                                check_samplerate=False)
+                        except ph5api.APIError as err:
+                            errors.add(err.msg)
+                            continue
```

Comment:
It seems like the cases where sample rate multiplier are 0 and sample rate multiplier is missing should act the same when it come to ph5availability and ph5toms. Likely, both cases should throw an error message and encourage the user to use `ph5/utilities/fix_srm.py` to fix them. Injecting a temporary number into a field seems confusing and doesn't really fix the problem in the data. 

> Some cases in ph5availability require sample_rate_multiplier_i in calculating, change code to use value 1 if it is missing.

This should probably not be automatic.

Reply:
As I stated at the description, in case of sample rate multiplier is missing we will consider its value is 1. The reason is I have no way to add a column to an existing ph5, unless I have to rebuild the whole ph5. I talked about that in the last meeting if you remembered. I know it doesn't really fix the problem in the data. but to handle this problem will be a lot of work. I tried that but was still not successful. We will talk about this again in the meeting today.

Reply:
If sample rate multiplier is missing than it seems practical to have the code assume that it is a value of 1. DAS.smr is very difficult to update so this issue needs to be dealt with in software rather than in the data.